### ISSUE TITLE
Struggling with connecting multiple 3.5 RPi displays 

### ISSUE THREAD

**Issue by cvels** [Bodmer/TFT_eSPI#1946]
BQLZR Red 240 x 320 resolutie 2,8 "SPI TFT-LCD-kleur Touch Panel Serial Port-module met PBC ILI9341 

https://www.megekko.nl/product/0/1708280/3-5-Touchscreen-Voor-Raspberry-Pi-ili9341-320-X-480?r=googlebeshopping&utm_source=googlebeshopping&utm_medium=cpc&gclid=Cj0KCQjw0JiXBhCFARIsAOSAKqCjmzzKeh8DgiM6DUhRFPtAKr0asZsiasgfc6TtfM_kBAFbr0LKB8UaAmCOEALw_wcB
https://gigatek.be/nl/velleman-3-5-touchscreen-voor-raspberry-pi-320-x-480--velvevmp400

http://www.lcdwiki.com/3.2inch_SPI_Module_ILI9341_SKU:MSP3218

5r9MgqnXjhirMt9

DOIT ESP32 DEVKIT V1

606
//	Serial.print ("******************HIER TC******************");

ssssif (scherm ==1)	{printf ("hallo") ;}

gith
canacon
vzeaua464ecb67k

Hi Bodmer,
I’m struggling with connecting multiple 3.5 displays to a NodeMCU ESP32 dev board.
I was only able to get it working when using 2 TFT SPI displays, based on the ILI9431 driver. (https://www.waveshare.com/wiki/2.4inch_LCD_Module )
The problem is that the ILI9431 (at least that is what I found on the net) is limited to a display size of something like 2.8inch.
My second attempt is to have 2 TFT SPI 3.5 inch displays also from Waveshare (https://www.waveshare.com/3.5inch-tft-touch-shield.htm ) working together on the same SPI bus. These displays are using driver chipset ILI9486 (RPI_ILI9486).
I’m using the CS trick to distinguish between the displays in the sketch, but for some reasons seems that the display is not picking up this.
At the moment, I’m only trying to have one display working …
When I define CS correctly in the setup11_RPi_touch_ILI9486.h, it’s all working. (I have tried different ways with user_setup.h where I defined RPI_ILI0486_Driver etc.
But when I define CS as -1 and do the CS handling in the sketch, the display stays white and not reacting at all.
In your code, I saw remarks about an RPi display is reacting slower/different on CS changes … maybe related.

IDE: Arduino
TFT_eSPI version: 2.4.72
interface SPI:

Any idea on  what I’m doing wrong?
Many thx in advance,
Kind regards,
Car
[test TFT.txt](https://github.com/Bodmer/TFT_eSPI/files/9242402/test.TFT.txt)
[Setup11_RPi_touch_ILI9486.txt](https://github.com/Bodmer/TFT_eSPI/files/9242414/Setup11_RPi_touch_ILI9486.txt)
l

**Comment by Bodmer** [comment]
I don't think this is an issue with the library is the display operates when CS is defined.

I would expect the display to operate if the CS is controlled by the sketch. Note that the chip select must be low during tft.init() or tft.begin().
